pymmmc/datasets/__init__.py

```diff
@@ -24,7 +25,9 @@ def cdnow_summary(**kwargs) -> pd.DataFrame:
     DataFrame
     """
 
-    return pd.read_csv("cdnow_summary.csv", **kwargs)
+    return pd.read_csv(
+        resource_filename("pymmmc", "datasets/cdnow_summary.csv"), **kwargs
+    )
```

Comment:
As per my previous comment about not using `pkg_resources`, the recommended way to do this is:
```suggestion
ref = importlib_resources.files("pymmmc") / "datasets" / "cdnow_summary.csv"
with importlib_resources.as_file(ref) as path:
        return pd.read_csv(path, **kwargs)
```